storage/buckets/get_bucket_policy.go

```diff
@@ -14,7 +14,7 @@
 
 package buckets
 
-// [START storage_get_bucket_policy]
+// [START storage_view_bucket_iam_members]
```

Comment:
Not sure we should change tags. If they are used already, the documentation samples will fail